client/src/router.ts

```diff
@@ -113,6 +115,18 @@ const routes = [
       image: understandLead.image,
     },
   },
+  {
+    path: SELECT_WATER_FILTER_ROUTE,
```

Comment:
add route attribute to BlogPost and do selectWaterFilter.route here instead of SELECT_WATER_FILTER_ROUTE

Reply:
done